### PR TITLE
METAL-2669 dbinstance chart fix, crd updates

### PR DESCRIPTION
**Pull request by hyunysmile**
* dbinstance chart fix: wrong `if end` close
* local minikube kubernetes version up
* crd some unnecessary required fields removed
* replace microk8s to k3d for integration test

### CODE REVIEW

**Review by dpeckett — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -35,9 +34,9 @@ To see more options of helm values [here](helm/README.md)
 * docker
 * [operator-sdk v0.13.0](https://github.com/operator-framework/operator-sdk/releases/tag/v0.13.0)
 * make
-* kubectl v1.14+(< v1.19)
```
Comment (dabde) [inline README.md:38]: kubectl can work over multiple versions, but best case to not drift more than 2 minor versions from your cluster.

(minor)

File: helm/db-operator/templates/crd.yaml [hunk 2]
```diff
@@ -317,63 +207,88 @@ spec:
...
+                bucket:
+                  type: string
               required:
-              - host
-              - port
+                - bucket
+            generic:
               properties:
+                backupHost:
+                  description: BackupHost address will be used for dumping database for backup Usually slave address for master-slave setup or cluster lb address If it's not defined, above Host will be used as backup host address.
```
Comment (dabde) [inline helm/db-operator/templates/crd.yaml:218]: replace master -> primary, slave -> seconday

**Review by dabde — COMMENTED** [review]
small comments.

File: README.md (hunk 1 above)
Comment (hyunysmile) [reply README.md:38]: client kubectl 1.20 works. I am gonna update docs with (< v1.21)

**Review by dabde — APPROVED** [review]

**Pull request merged**